groovy/gradle/src/org/netbeans/modules/gradle/GradleDistributionManager.java

```diff
@@ -188,8 +188,10 @@ public NbGradleVersion evaluateGradleWrapperDistribution(File rootDir) {
         List<NbGradleVersion> ret = new ArrayList<>();
         JSONParser parser = new JSONParser();
         try {
-            URL allVersions = new URL("http://services.gradle.org/versions/all"); //NOI18N
-            try (InputStreamReader is = new InputStreamReader(allVersions.openStream())) {
+            URL allVersions = new URL("https://services.gradle.org/versions/all"); //NOI18N
+            HttpsURLConnection conn = (HttpsURLConnection) allVersions.openConnection();
+            conn.setInstanceFollowRedirects(true);
+            try (InputStreamReader is = new InputStreamReader(conn.getInputStream())) {
```

Comment:
I would add the charset to the InputStreamReader. There are still JDKs out there, that don't use UTF-8 by default and in the future the default might change. `java.nio.charset.StandardCharsets.UTF-8` would be the value I'd use.

Reply:
Added that. Thanks!